CustomerOrderProduct/CustomerOrderProduct/Controllers/OrderController.cs

```diff
@@ -1,44 +1,119 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using CustomerOrderProduct.DTOS;
+using CustomerOrderProduct.Interfaces;
+using Microsoft.AspNetCore.Mvc;
 
 namespace CustomerOrderProduct.Controllers
 {
 	[ApiController]
 	[Route("[controller]")]
 	public class OrderController : ControllerBase
 	{
+		private readonly IOrderService _orderService;
+
+		public OrderController(IOrderService orderService)
+		{
+			_orderService = orderService;
+		}
+
 		[HttpGet]
 		[Route("getAll")]
 		public async Task<ActionResult> GetAll()
 		{
-			return null;
+			var result = await _orderService.GetOrders();
+
+			if (result.ErrorCode == "404")
```

Comment:
You can avoid hardcoding these codes. You can use the StatusCodes class
![image](https://user-images.githubusercontent.com/5354994/202875518-cd77cea9-74e0-48f1-acd6-861acb1f31d7.png)